docs_site/source/topic.html.erb

```diff
@@ -0,0 +1,111 @@
+---
+title: Home
```

Comment:
is this file needed?

Reply:
@kennyadsl No, it's not. It was just a template/example of the styles we are using. I'll remove it!